### ISSUE TITLE
Create shortcuts for the whole rules in stories

### ISSUE THREAD

**Issue by Ghostvv** [RasaHQ/rasa#6486]
in order to successfully train TED on the full stories, the rules should be present in the stories. We can use rule snippets names (in the same spirit as checkpoints) where instead writing all events inside the rule in the story, users could just add a rule name and in `generator.py` they will be substituted with actual events from rules

**Comment by wochinge** [comment]
@akelad Why did you add this to the RC milestone? Seems like nice-to-have and something which could go into 2.1

**Comment by akelad** [comment]
why a nice to have?

**Comment by wochinge** [comment]
@akelad I think it's useful but as Maxime said I don't think it's critical for the release. Plus I think some user feedback on this would be great before starting to implement a solution.

**Comment by Ghostvv** [comment]
> Plus I think some user feedback on this would be great before starting to implement a solution.
how would we get user feedback for a feature that is not implemented?

**Comment by madanmeena** [comment]
@Ghostvv  I was using rules for feedback which in our case is a fixed turn of event as follows
`- rule: user positive_feedback
  steps:
  - action: utter_feedback_1_options
  - intent: positive_feedback
    entities:
    - feedback: true
  - slot_was_set:
    - feedback: true
  - action: action_feedback
  - action: utter_feedback_2_thanks
- rule: user negative_feedback
  steps:
  - action: utter_feedback_1_options
  - intent: negative_feedback
    entities:
    - feedback: false
  - slot_was_set:
    - feedback: false
  - action: action_feedback
  - action: utter_feedback_2_thanks
  `
 In my stories conversation where we ask for feedback ends with utter_feedback_1_options.   I was expecting that it should be enough but it was not, so I ended up adding a few examples of feedback in actual stories and things started working, so it would be great if we could add a checkpoint feature so that we are not duplicating our code.

**Comment by Ghostvv** [comment]
also, we need to make sure, that it doesn't clash with https://github.com/RasaHQ/rasa/issues/7226
